src/emitter.cpp

```diff
@@ -770,6 +774,21 @@ const char* Emitter::ComputeFullBoolName(bool b) const {
                          // these answers
 }
 
+const char* Emitter::ComputeNullName() const {
+  switch (m_pState->GetNullFormat()) {
+    case LowerNull:
+      return "null";
+    case UpperNull:
+      return "NULL";
+    case CamelNull:
+      return "Null";
+    case TildeNull:
+      return "~";
+    default:  // fallthrough
```

Comment:
What I meant was:

```
case TildeNull:
  // fallthrough
default:
  return "~";
```

This is the same as the previous behavior you had, but with the comment.

What you have now would not return any value in the default case, which would be undefined behavior.

Reply:
haha, I misunderstanded your meaning.